### PR TITLE
apply setup-py-upgrade

### PR DESCRIPTION
**Pull request by graingert**
https://github.com/asottile/setup-py-upgrade

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,30 @@
+[metadata]
+name = sshim
+version = 1.3.2
```
Comment (graingert) [inline setup.cfg:3]:
```suggestion
version = 1.3.3
```

File: setup.cfg [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+[options]
+install_requires =
+    paramiko
+    six
+zip_safe = True
+include_package_data = False
+packages = find:
+
+[options.extras_require]
+tests = nose
```
Comment (graingert) [inline setup.cfg:27]:
```suggestion
tests = pytest
```

**Comment by graingert** [comment]
@simon-engledew fyi you're getting some warnings:

```

  /opt/hostedtoolcache/Python/3.9.6/x64/lib/python3.9/site-packages/_pytest/threadexception.py:75: PytestUnhandledThreadExceptionWarning: Exception in thread sshim.Actor(0)
  
  Traceback (most recent call last):
    File "/opt/hostedtoolcache/Python/3.9.6/x64/lib/python3.9/threading.py", line 973, in _bootstrap_inner
      self.run()
    File "/home/runner/work/sshim/sshim/sshim/Server.py", line 260, in run
      self.channel.send_exit_status(0)
    File "/opt/hostedtoolcache/Python/3.9.6/x64/lib/python3.9/site-packages/paramiko/channel.py", line 425, in send_exit_status
      self.transport._send_user_message(m)
    File "/opt/hostedtoolcache/Python/3.9.6/x64/lib/python3.9/site-packages/paramiko/transport.py", line 1863, in _send_user_message
      self._send_message(data)
    File "/opt/hostedtoolcache/Python/3.9.6/x64/lib/python3.9/site-packages/paramiko/transport.py", line 1839, in _send_message
      self.packetizer.send_message(data)
    File "/opt/hostedtoolcache/Python/3.9.6/x64/lib/python3.9/site-packages/paramiko/packet.py", line 431, in send_message
      self.write_all(out)
    File "/opt/hostedtoolcache/Python/3.9.6/x64/lib/python3.9/site-packages/paramiko/packet.py", line 367, in write_all
      raise EOFError()
  EOFError
  
    warnings.warn(pytest.PytestUnhandledThreadExceptionWarning(msg))
```

you can enable filterwarnings = error in:

```toml
# pyproject.toml
[tool.pytest.ini_options]
filterwarnings = [
    "error",
]
```

**Comment by simon-engledew** [comment]
> you can enable filterwarnings = error to surface these in:

🤔 Weirdly that changes nothing for me locally. I did try and fix the threading issues about a year ago but I decided that it was easier to just [move everything onto aio](https://github.com/simon-engledew/aiosshim).

Sorry for taking so long to get to this. 😞

**Pull request merged**